vmdb/app/controllers/ems_common.rb

```diff
@@ -219,11 +219,11 @@ def form_field_changed
           # Hide/show C&U credentials tab
           page << "$('metrics_li').#{params[:server_emstype] == "rhevm" ? "show" : "hide"}();"
         end
-        if ["openstack"].include?(params[:server_emstype])
+        if ["openstack", "openstack_infra"].include?(params[:server_emstype])
```

Comment:
I really don't like this tests. Neither in the original code, nor here.

I would prefer some list of properties per provider type and testing against this list. So that adding new provider type would mean defining a list of properties for that provider and NOT changing all these `include` and `==` tests.